static/docs/commands-reference/pull.md

```diff
@@ -1,53 +1,123 @@
+files which have not changed.
+
+If the `--all-branches` or `--all-tags` options are specified, `dvc pull` will
+examine files associated with all branches or tags of the DVC files in
+the project directory.
+
+If one or more _target_'s are specified, DVC only considers the files associated
+with those stages.  Using the `--with-deps` option DVC tracks dependences
+backward through the pipeline to find files to pull.
+
+??? --force
+
+With the `--recursive dirname` option DVC searches the named directory and its
+subdirectories for files to download.
 
 After data file is in cache DVC utilizes OS specific mechanisms like reflinks or
 hardlinks to put it into the working space without copying. See `dvc checkout`
 for more details.
 
-See `dvc remote`, `dvc config` and
-[remote storages](https://dvc.org/doc/get-started/configure)
-for more information on how to configure the remote storage.
+## Options
 
-```usage
-    usage: dvc pull [-h] [-q] [-v] [-j JOBS] [-r REMOTE] [-a]
-                    [-T] [-d] [-f]
-                    [targets [targets ...]]
+* `--show-checksums` Show checksums instead of file names.
 
-    positional arguments:
-      targets               DVC files.
+* `-r REMOTE`, `--remote REMOTE` specifies which remote cache 
+  (see `dvc remote list`) to pull from. The argument, `REMOTE`, is a
+  remote name defined using the `dvc remote add` command.
 
-    optional arguments:
-      -h, --help            show this help message and exit
-      -q, --quiet           Be quiet.
-      -v, --verbose         Be verbose.
-      -j JOBS, --jobs JOBS  Number of jobs to run simultaneously.
-      --show-checksums      Show checksums instead of file names.
-      -r REMOTE, --remote REMOTE
-                            Remote repository to pull from
-      -a, --all-branches    Fetch cache for all branches.
-      -T, --all-tags        Fetch cache for all tags.
-      -d, --with-deps       Fetch cache for all dependencies of the specified
-                            target.
-      -f, --force           Do not prompt when removing working directory files.
-      -R RECURSIVE, --recursive RECURSIVE
-                            Pull cache for subdirectories of the specified
-                            directory.
-```
+* `-a`, `--all-branches`  Determines the files to download by comparing across
+  all branches of the workspace.
+
+* `-T`, `--all-tags` Determines the files to download by comparing across
+  all tags of the workspace.
+
+* `-d`, `--with-deps` Determines the files to download by searching backwards
+  in the pipeline from the named stage(s).  The only files which will be
+  considered are associated with the named stage, and the stages which execute
+  earlier in the pipeline.
+
+* `-f`, `--force` Do not prompt when removing working directory files.
+
+* `-R dirname`, `--recursive dirname` Determines the files to download by
+  searching the named directory and its subdirectories for changed files.
```

Comment:
this option description should be updated, we are searching for DVC files a believe

Reply:
Are you referring to `--force`?  That text is directly from the help output in the command.

As I said elsewhere I was unable to work out what `--force` is supposed to do in the first place.

Reply:
@efiop @mroutis @pared could you guys help to figure this out please?

Reply:
I initially meant -R. The phrase `... for changed files` is not clear. It is just simply looking for all DVC files inside the directory, downloads and checkouts missing references into cache

Reply:
@shcheklein I didn't do any task related to force yet, but for what I see in tests and code it seems that what force does is simply remove outs before checking them out, without prompting user whether dvc should remove it.

Reply:
@pared makes sense, I just want to confirm that it is asking this if an existing file has an older/different version than the one referenced in the dvc file

Reply:
Yes, prompt is displayed upon finding existing file that is referenced in stage file, that is different than cached one.

Reply:
@pared `different than cached one` -> different than the one in the working space`, right? :)

Reply:
Correct, sorry, I have to be careful with words :)